tests/functional/f/forgotten_debug_statement_py37.py

```diff
@@ -0,0 +1,10 @@
+# pylint: disable=missing-docstring
+
+import sys
+
+breakpoint()  # [forgotten-debug-statement]
+sys.breakpointhook()  # [forgotten-debug-statement]
```

Comment:
Can you add `pdb.set_trace()` here as well?

Reply:
Shit, totally forgot about this! Doing this now